src/Chip/chip.scss

```diff
@@ -0,0 +1,28 @@
+@import "../../node_modules/@sebgroup/bootstrap/scss/variables";
+
+.custom-chip {
+    display: inline-flex;
+    align-items: center;
+    justify-content: space-between;
+    border-radius: 4px;
+    background-color: $gray-200;
+    height: 32px;
+    padding: 0 8px;
+    font-size: $font-size-base;
+    &:hover {
+        background-color: $gray-100;
+        > .chip-close {
+            color: $red;
+        }
+    }
+    > .chip-close {
+        cursor: pointer;
+        margin-left: 10px;
```

Comment:
Design Library:
![image](https://user-images.githubusercontent.com/49055952/79089070-4dea4380-7d77-11ea-9c35-3ca087ecdd1f.png)

Your component:
![image](https://user-images.githubusercontent.com/49055952/79089108-74a87a00-7d77-11ea-9c51-5395b1fd291c.png)

Comparing the two I noticed the design library has the same distance between the content and the X button as the padding, so you should change this `margin-left` to `8px` since that is your padding. Otherwise the distance between the X and the content looks too big

Reply:
added